libcnb-data/src/buildpack/stack.rs

```diff
@@ -48,3 +48,69 @@ pub enum StackError {
     #[error("Invalid Stack ID: {0}")]
     InvalidStackId(#[from] StackIdError),
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn deserialize_specific_stack_without_mixins() {
+        let toml_str = r#"
+id = "heroku-20"
+"#;
+        assert_eq!(
+            toml::from_str::<Stack>(toml_str),
```

Comment:
The existing TOML parsing tests use `raw` as the variable name here, however I'm not that keen on it, since it's a bit vague (eg `toml::from_str::<Stack>(raw)`). As part of cleaning up the tests I was going to standardise on something else. Open to ideas :-)